tests/run_test_within_docker.sh

```diff
@@ -22,7 +22,7 @@ if [ $# -gt 1 ]; then
   case "$2" in
     valgrind)
       export DEBIAN_FRONTEND=noninteractive
-      apt install -y valgrind
+      apt-get update && apt install -y valgrind
```

Comment:
Should we put `-y` after update also?

Reply:
No, that not necessary. For "apt install" the -y option is necessary if you install packages in a script, but for update we don't need it.